logappender/src/main/java/org/apache/rocketmq/logappender/common/ProducerInstance.java

```diff
@@ -39,7 +39,7 @@
 
     public static final String DEFAULT_GROUP = "rocketmq_appender";
 
-    private static ConcurrentHashMap<String, MQProducer> producerMap = new ConcurrentHashMap<String, MQProducer>();
```

Comment:
There is log4j2 implementation in logappender.Since log4j2 not supported jdk 1.6,logappender do not need to be compatible.